Tools/wasm/wasi/__main__.py

```diff
@@ -127,18 +134,19 @@ def build_python_path():
     if not binary.is_file():
         binary = binary.with_suffix(".exe")
         if not binary.is_file():
-            raise FileNotFoundError("Unable to find `python(.exe)` in "
-                                    f"{BUILD_DIR}")
+            raise FileNotFoundError("Unable to find `python(.exe)` in " f"{BUILD_DIR}")
```

Comment:
I think the whole string (from `"Unable...` on) can be concatenated and made an fstring here if I'm not mistaken, surprised black didn't do this.
```suggestion
            raise FileNotFoundError(f"Unable to find `python(.exe)` in {BUILD_DIR}")
```